### PR TITLE
docs: Component Styles guide for CLI

### PR DESCRIPTION
**Pull request by wardbell**
Issue #19510 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by Foxandxss — APPROVED** [review]

File: aio/content/examples/component-styles/plnkr.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "files": [
     "!**/*.d.ts",
     "!**/*.js",
-    "!**/*.native.*"
+    "!**/*.native.*",
+    "!**/*.[1].*"
```
Comment (gkalpak) [inline aio/content/examples/component-styles/plnkr.json]: OOC, isn't `[1]` the same as `1`?

File: aio/content/guide/component-styles.md [hunk 2]
```diff
@@ -165,110 +149,119 @@ There are several ways to add styles to a component:
...
+### Styles in component metadata
 
 You can add a `styles` array property to the `@Component` decorator.
-Each string in the array (usually just one string) defines the CSS.
 
+Each string in the array defines some CSS for this component.
 
-<code-example path="component-styles/src/app/hero-app.component.ts" title="src/app/hero-app.component.ts">
-
+<code-example path="component-styles/src/app/hero-app.component.ts" title="src/app/hero-app.component.ts (CSS inline">
```
Comment (gkalpak) [inline aio/content/guide/component-styles.md]: `(CSS inline` --> `(CSS inline)`

File: aio/content/examples/component-styles/plnkr.json (hunk 1 above)
Comment (Foxandxss) [reply aio/content/examples/component-styles/plnkr.json]: maybe, but we are using that syntax everywhere because sometimes we have more values than `1`

**Comment by vicb** [comment]
missing:
- target label
- merge label
- green CI

**Pull request merged**